pkg/reconciler/base.go

```diff
@@ -36,6 +36,11 @@ import (
 	"knative.dev/pkg/logging/logkey"
 )
 
+// ConfigStore is a minimal interface to the config stores used by our controllers.
+type ConfigStore interface {
```

Comment:
WDYT about using https://godoc.org/knative.dev/serving/pkg/reconciler#ConfigStore instead?

Reply:
NVM: It looks like that would cause module problems.